misc/logging/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+use tracing_subscriber::EnvFilter;
+
+pub fn init_tracing_subscriber_with_default_env_filter() {
+    init_tracing_subscriber_with_env_filter(EnvFilter::from_default_env());
+}
+
+pub fn init_tracing_subscriber_with_env_filter(filter: impl Into<EnvFilter>) {
```

Comment:
```suggestion
pub fn init_tracing_subscriber_with_env_filter(filter: EnvFilter) {
```

Nit: Is there any type that implements `Into<EnvFilter>` other than `EnvFilter` itself?

Reply:
> Nit: Is there any type that implements `Into<EnvFilter>` other than `EnvFilter` itself?

yes, string and string literals like "my_crate=info,my_crate::my_mod=debug,[my_span]=trace".